config/locales/cy.yml

```diff
@@ -308,9 +308,9 @@ cy:
     accessibility: Hygyrchedd
     accessibility_link: /cy/corporate/hygyrchedd
     privacy_statement: Rhybudd preifatrwydd
-    privacy_statement_link: /cy/corporate/polisipreifatrwydd
+    privacy_statement_link: https://www.moneyhelper.org.uk/cy/privacy
```

Comment:
Not a big deal but would it be better to keep the relative URLs here since this will be on the moneyhelper domain (e.g. `/cy/privacy`)?

Reply:
> Not a big deal but would it be better to keep the relative URLs here since this will be on the moneyhelper domain (e.g. `/cy/privacy`)?

Oh wait, that's not how this works is it? this remains under the MAS domain. In that case ignore that ☝️